blog/app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    has_secure_password
+    acts_as_voter
+    # attr_accessor :username, :email, :password, :password_confirmation
+
```

Comment:
You can use **has_many :post**  as a user can do multiple posts or wherever you calling it.